src/main/java/com/akhil/app/controllers/HealthController.java

```diff
@@ -0,0 +1,20 @@
+package com.akhil.app.controllers;
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthController {
+
+    @GetMapping("/healthup")
+    public String healthUp(){
+        return ("{\"Status\":\"Up\"}");
+    }
+
+    @GetMapping("/healthdown")
```

Comment:
Health down is not necessary, if the healtUp is not responding it means health is down

Reply:
Made changes, thanks